nuxeo-drive-client/nxdrive/engine/engine.py

```diff
@@ -1005,3 +1009,29 @@ def fetch_all_users(self):
         except Exception as e:
             log.exception(e)
         return
+
+    def refresh_user(self, user_id):
```

Comment:
Another interesting way to address this caching (in _user_cache), if needed, is to use the lru_cache decorator:
@functools.lru_cache()
def refresh_user(self, user_id)
  ...